drivers/serial/Kconfig.stm32

```diff
@@ -5,6 +5,7 @@
 
 DT_COMPAT_ST_STM32_DMA_V1 := st,stm32-dma-v1
 DT_COMPAT_ST_STM32_DMA_V2 := st,stm32-dma-v2
+DT_COMPAT_ST_STM32_DMA_V2BIS := st,stm32-dma-v2bis
```

Comment:
Please move this in a dedicated commit

Reply:
removing all the _tests_ stuff

Reply:
In case there is a element missing in the dma-cells, the building will fail with something like:
```
nucleo_f746zg.dts.pre.tmp:2550.2-2551.22: Warning (dmas_property): /soc/spi@40013000:dmas: cell 5 is not a phandle reference
nucleo_f746zg.dts.pre.tmp:295.22-304.5: Warning (dmas_property): /soc/spi@40013000: Missing property '#dma-cells' in node /clocks/pll or bad phandle (referred from dmas[5])
  also defined at nucleo_f746zg.dts.pre.tmp:2411.20-23
  also defined at nucleo_f746zg.dts.pre.tmp:2504.7-2508.3
  also defined at nucleo_f746zg.dts.pre.tmp:2549.7-2553.3
devicetree error: <Node /clocks/pll in 'nucleo_f746zg.dts.pre.tmp'> lacks #dma-cells

```

Reply:
ok, then we'll need to send a mail to users & dev list to inform of this change.

Reply:
What about the case when there is one too much ?

Reply:
in case there is an extra parameter, the error comes in the building as well:
```
stm32f3_disco.dts.pre.tmp:1515.2-1516.22: Warning (dmas_property): /soc/spi@40013000:dmas: cell 3 is not a phandle reference
stm32f3_disco.dts.pre.tmp:1515.2-1516.22: Warning (dmas_property): /soc/spi@40013000:dmas: Could not get phandle node for (cell 3)
devicetree error: dma-names property in /soc/spi@40013000 in stm32f3_disco.dts.pre.tmp has 2 strings, expected 4 strings
```